src/pages/MyKiva/MyKivaPageContent.vue

```diff
@@ -672,15 +672,19 @@ export default {
 }
 
 :deep(.bp-sidesheet-wrapper > div) {
-	max-width: 100% !important;
+	width: 100% !important;
 	min-height: 100vh;
 
-	@screen md {
-		max-width: 50% !important;
-	}
+	@media (width >= 768px) {
+		width: 50% !important;
+    }
 
-	@screen xl {
-		max-width: 600px !important;
-	}
+	@media (width >= 1280px) {
+		width: 600px !important;
+    }
+}
+
+:deep(.bp-sidesheet-wrapper #sidesheet-content) {
+	min-height: 33vh;
```

Comment:
I'm not sure about this one, why is it needed?

Reply:
Loan id is prefetched, BP Side Sheet is fetching data on mounted from composable.

Reply:
Yeah fetching from cache and loaders just for a moment.
Issue with bp content is that heigh is calculated dynamically based on screen size to show loan CTA floating at bottom of the screen, something we can't do on server side rendering.
If we want a different approach, that is possible, we should refactor shared component, data fetching and ensure all pages we are using this BP side sheet works fine.
Probably this is the simplest solution as just needed at mykiva page, worth testing,